libraries/chain/market_evaluator.cpp

```diff
@@ -406,6 +399,22 @@ void_result bid_collateral_evaluator::do_evaluate(const bid_collateral_operation
    else
        FC_ASSERT( o.debt_covered.amount > 0, "Can't find bid to cancel?!");
 
+   if( o.additional_collateral.amount > 0 )
+   {
+      if( _bid && d.head_block_time() >= HARDFORK_CORE_1692_TIME )
+      {
+         asset delta = o.additional_collateral - _bid->get_additional_collateral();
+         FC_ASSERT( d.get_balance(*_paying_account, _bitasset_data->options.short_backing_asset(d)) >= delta,
+                    "Cannot increase bid from ${oc} to ${nc} collateral when payer only has ${b}",
+                    ("oc", _bid->get_additional_collateral().amount)("nc", o.additional_collateral.amount)
+                    ("b", d.get_balance(*_paying_account, o.additional_collateral.asset_id(d)).amount) );
+      } else
```

Comment:
I guess this (and the hf check) can be removed after the hard fork? Maybe add a TODO comment.

Reply:
Added.